packages/cli/src/commands/build.ts

```diff
@@ -21,7 +21,9 @@ export const buildOptions = (yargs: CommonYargsArgv) =>
     .option("template", {
       type: "string",
       default: "vercel",
-      describe: "[Experimental] Template to use for the build",
+      describe: `[Experimental] Template to use for the build
+Templates Available
+${PROJECT_TEMPALTES.join(`\n`)}`,
```

Comment:
You can try `choices: ['xs', 's', 'm', 'l', 'xl']` field

Reply:
We can't use the `choices` because local templates will fail the check. As `yargs` itself is throwing error if users pass
```sh
webstudio-cli build --template ./template
```

something like this.